ttt-lexical-patterns.lisp

```diff
@@ -9,6 +9,29 @@
 (defparameter *tense* '(past pres cf))
 (defparameter *coordinator* '(and or but because))
 (defparameter *detformer* '(nquan fquan))
+(defparameter *semtypes*
+  '(("([A-Z]|_)*\\.PRO" . "D")
+    ("\\|([A-Z]|_)*\\|" . "D") ; doesn't work properly. Does work with '\|' instead of '|' in the symbol passed to atom-semtype? I think CL doesn't allow | in symbols.
+    ("([A-Z]|_)*\\.P" . "D => (D => (S => 2))")
+    ("([A-Z]|_)*\\.PS" . "(S => 2) => ((S => 2) => (S => 2))")
+    ("([A-Z]|_)*\\.N" . "D => (S => 2)")
+    ("([A-Z]|_)*-OF\\.N" . "D => (D => (S => 2))")
+    ("([A-Z]|_)*\\.A" . "D => (S => 2)") ; doc gives two possibilities for this
```

Comment:
Just keep this comment for now. The next step is going to be to convert these type strings to data structures, so we'll make sure to deal with that. I just haven't come up with good notation for this yet. Maybe `{(D => (S => 2))|(D => (D => (S => 2)))}`? Which is a generalization of the alternatives notation we're already using for syntactic categories. Anyway, that's not really that important.